root/utility/tableColumns.js

```diff
@@ -294,14 +421,20 @@ export const seriesOrderingTypeColumn:
     accessor: 'orderingTypeID',
   };
 
-
 export const subscriptionColumn:
   ColumnOptions<{+subscribed: boolean, ...}, boolean> = {
     Cell: ({cell: {value}}) => yesNo(value),
     Header: N_l('Subscribed'),
     accessor: 'subscribed',
   };
 
+export const taggerColumn:
+  ColumnOptions<{+subscribed: boolean, ...}, boolean> = {
```

Comment:
The `+subscribed` part of the type seems wrong.

Reply:
Oh, yes. I changed it to ColumnOptions<RecordingT | ReleaseT, RecordingT | ReleaseT> but I'm actually not sure what the second part should even be (I don't have an accessor at all here).

Reply:
`ColumnOptions<RecordingT | ReleaseT, void>` would probably be just fine.